CHANGELOG.md

```diff
@@ -18,7 +18,8 @@ Current
 
 
 #### Changed:
-
+- [ISSUE-17](https://github.com/yahoo/fili/issues/17) [Added pagination parameters to PreResponse] (https://github.com/yahoo/fili/pull/19) 
```

Comment:
In the future, if there isn't already an issue for something you're opening a PR for, don't bother with opening an issue as well. In GitHub, PRs _are_ issues, so there's no need to create an additional artifact.